client/scss/components/_tabs.scss

```diff
@@ -111,53 +9,71 @@
   }
 }
 
-@include media-breakpoint-up(sm) {
-  .tab-nav {
-    // For cases where tab-nav should merge with header
-    &.merged {
-      background-color: $color-header-bg;
-    }
-
-    li {
-      width: auto;
-      padding: 0;
-    }
-
-    a {
-      padding-inline-start: $mobile-nice-padding;
-      padding-inline-end: $mobile-nice-padding;
-    }
-
-    li.settings a {
-      padding-inline-start: 2em;
-      padding-inline-end: 2em;
+.w-tabs {
+  &__wrapper {
+    @apply w-mb-10;
+  }
+
+  &__list {
+    @apply w-flex w-space-x-6 w-border-b w-border-grey-100 w-w-fit w-px-[20px] sm:w-px-[40px] md:w-px-[80px];
+  }
+
+  &__tab {
+    @apply w-label-3
+    w-outline-offset-inside
```

Comment:
For me this results in cropping of the tab text, so would be better with the default offset.

```suggestion
```